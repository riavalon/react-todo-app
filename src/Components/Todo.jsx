import React from 'react'
import styles from '../Styles/Todo.module.css'

const Todo = props => {
  return (
    <li 
      className={styles.listItem} 
      key={props.id} 
      onClick={() => props.onClick(props.id)}
    >
      {props.text}
    </li>
  )
}
export default Todo