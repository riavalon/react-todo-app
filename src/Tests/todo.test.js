import React from 'react';
import renderer from 'react-test-renderer';
import Todo from '../Components/Todo';

it('renders correctly', () => {
  const tree = renderer.create(
    <Todo 
      text="Todo or not Todo, that is the question."
    />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});