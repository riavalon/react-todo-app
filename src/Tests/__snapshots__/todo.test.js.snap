// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<li
  className="listItem"
  onClick={[Function]}
>
  Todo or not Todo, that is the question.
</li>
`;
